pkg/etcdutil/etcdutil_test.go

```diff
@@ -21,43 +21,35 @@ import (
 	"testing"
 	"time"
 
-	. "github.com/pingcap/check"
+	"github.com/stretchr/testify/require"
 	"go.etcd.io/etcd/clientv3"
 	"go.etcd.io/etcd/embed"
 	"go.etcd.io/etcd/pkg/types"
 )
 
-func Test(t *testing.T) {
-	TestingT(t)
-}
-
-var _ = Suite(&testEtcdutilSuite{})
-
-type testEtcdutilSuite struct{}
-
-func (s *testEtcdutilSuite) TestMemberHelpers(c *C) {
+func TestMemberHelpers(t *testing.T) {
 	cfg1 := NewTestSingleConfig()
 	etcd1, err := embed.StartEtcd(cfg1)
 	defer func() {
 		etcd1.Close()
 		CleanConfig(cfg1)
 	}()
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
```

Comment:
How about use `re:=require.New(t);re.NoError` if there are many assert in one unit test?

Reply:
Good idea!

Reply:
What about rewriting this way in another PR? Let's merge this one first.